### PR TITLE
Change the order of available quality options

### PR DESCRIPTION
**Pull request by jdragonbae**
This patch changes the order of available quality options. `auto` option would
be default option when play a new media on YouTube. So, it should be top among
the available quality options.

Refs: #776

### CODE REVIEW

File: src/org/triple/banana/util/YouTubeUtil.java [hunk 1]
```diff
@@ -187,6 +187,12 @@ public static void dismissQualityOptionsDialog() {
         try {
             JSONObject json = new JSONObject(jsonString);
             JSONArray availableQualities = json.getJSONArray("availableQualities");
+            for (int i = availableQualities.length() - 1; i > 0; i--) {
+                if (availableQualities.getString(i).equals("auto")) {
+                    availableQualities.put(0, availableQualities.remove(i));
```
Comment (romandev) [inline src/org/triple/banana/util/YouTubeUtil.java:192]: This is O(n) operation.

File: src/org/triple/banana/util/YouTubeUtil.java [hunk 2]
```diff
@@ -187,6 +187,12 @@ public static void dismissQualityOptionsDialog() {
         try {
             JSONObject json = new JSONObject(jsonString);
             JSONArray availableQualities = json.getJSONArray("availableQualities");
+            for (int i = availableQualities.length() - 1; i > 0; i--) {
```
Comment (romandev) [inline src/org/triple/banana/util/YouTubeUtil.java:190]: The `auto` element might be first element but the loop is anyway O(n).
Also, The `availableQualities.put(0, availableQualities.remove(i));` is O(n).
So, your changes are O(n^2).

Comment (jdragonbae) [reply src/org/triple/banana/util/YouTubeUtil.java]: Ack. Thanks for the review!

File: src/org/triple/banana/util/YouTubeUtil.java (hunk 1 above)
Comment (jdragonbae) [reply src/org/triple/banana/util/YouTubeUtil.java]: Ack.

**Review by romandev — APPROVED** [review]
lgtm

**Pull request merged**